modules/ROOT/pages/connected-apps-overview.adoc

```diff
@@ -0,0 +1,21 @@
+= Connected Apps
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Connected Apps provide a framework that enables an external application to integrate with MuleSoft using APIs via standard protocols, such as SAML, OAuth, and OpenID Connect. Connected apps help user delegate their access without sharing sensitive credentials or giving full control of their account to third parties. Actions taken by connected apps are auditable, and you can also revoke access at any time.
```

Comment:
nit: Connected apps help user -> Connected apps help users. Also the paragraph switches from the third person to the second person